slack-plugin/src/main/kotlin/slack/gradle/SlackProperties.kt

```diff
@@ -332,6 +332,14 @@ public class SlackProperties private constructor(private val project: Project) {
   public val autoApplyCacheFix: Boolean
     get() = booleanProperty("slack.auto-apply.cache-fix", defaultValue = true)
 
+  /* Detekt configs. */
+  /** Detekt config files, evaluated from rootProject.file(...). */
```

Comment:
Wasn't sure if this comment was intended
```suggestion
  /** Detekt config files, evaluated from rootProject.file(...). */
```

Reply:
The first one is a block comment about the multiple below props